test/controllers/ImporterEORINumberControllerSpec.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.ControllerSpecBase
+import controllers.actions.FakeDataRetrievalAction
+import forms.ImporterEORINumberFormProvider
+import mocks.repositories.MockSessionRepository
+import models.UserAnswers
+import pages.ImporterEORINumberPage
+import play.api.http.Status
+import play.api.mvc.{AnyContentAsFormUrlEncoded, Result}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{charset, contentType, defaultAwaitTimeout, redirectLocation, status}
+import views.html.ImporterEORINumberView
+
+import scala.concurrent.Future
+
+class ImporterEORINumberControllerSpec extends ControllerSpecBase {
+
+  val userAnswersWithImporterEORINumber: Option[UserAnswers] = Some(UserAnswers("some-cred-id")
+    .set(
+      ImporterEORINumberPage, "GB345834921000"
```

Comment:
"GB345834921000" is duplicated quite a bit throughout could you make it a variable and re-use it